packages/fxa-customs-server/lib/reputationService.js

```diff
@@ -51,13 +51,18 @@ var get = function (log, ipClient, ip) {
         log.info({ op: 'fetchIPReputation', ip: ip, err: 'Reputation not found for IP.',
           rtime: getRequestTime(response) })
       } else {
-        var err = { status: response.statusCode, body: response.body }
-        log.error({ op: 'fetchIPReputation', ip: ip, err: err, rtime: getRequestTime(response) })
+        log.error({
+          err: response.body,
+          ip,
+          op: 'fetchIPReputation',
+          rtime: getRequestTime(response),
+          statusCode: response.statusCode,
```

Comment:
statusCode is also [used up here](https://github.com/mozilla/fxa/pull/809/files#diff-1c4367a4c81cd2d8ce8b37d310d13972R33), so I tried to stay consistent.